packages/iris-grid/src/sidebar/visibility-ordering-builder/VisibilityOrderingBuilder.tsx

```diff
@@ -1067,8 +1227,9 @@ class VisibilityOrderingBuilder extends Component<
           <SearchInput
             className="visibility-search"
             value={searchFilter}
-            matchCount={searchFilter ? selectedColumns.size : undefined}
+            matchCount={searchFilter ? matchCount : undefined}
             onChange={this.handleSearchInputChange}
+            queryParams={queryParams}
```

Comment:
i thought passing in queryParams would be nicer than adding 2 optional params into SearchInput. Open to doing the other method though